app/src/main/java/dadeindustries/game/gc/mechanics/turn/Event.java

```diff
@@ -0,0 +1,33 @@
+package dadeindustries.game.gc.mechanics.turn;
+
+import dadeindustries.game.gc.model.Coordinates;
+
+public class Event {
+	public enum EventType {
+		BATTLE, RANDOM_EVENT
+	}
+
+	private EventType type;
+	private String description;
+	private Coordinates coordinates;
+
+	public Event(EventType type, String description, Coordinates coods) {
+		this.description = description;
+		this.type = type;
+		this.coordinates = coods;
+	}
+
+	public EventType getEventType() {
+		return type;
+	}
+
+	public String getDescription() {
+		return description;
+	}
+
+	public void appendDescription(String string) { description = description + "\n* " + string; };
```

Comment:
This should be over 3 lines, also that final semicolon on the end is unnecessary and should be removed. Autoformatting this file is a good idea.